starter-code/views/movies-view/movies-list.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+    <h2>List of Films</h2>
+
+    <ul>
+        {{#each allMovies}}
+        <li class="celeb-list">{{title}} <small>
+                <a class="text-info" href="/movies/detalle/{{_id}}">details</a>
+                <a class="text-info" href="/movies/eliminar-movie?movies_id={{_id}}">delete</a>
```

Comment:
```suggestion
 <form method="POST" action="/movies/eliminar-movie?movies_id={{_id}}">
                <button>delete</button>
 </form>
```
Para hacerlo con el verbo HTTP _POST_, podrías haber envuelto el ancle en un formulario con este método y el  action que corresponda, en todo caso no está mal, iréis trabajando sobre esto a lo largo de la semana.